.github/workflows/ci.yml

```diff
@@ -535,6 +535,64 @@ jobs:
         with:
           name: behavex_output
           path: build/behavex_output
+  modular_docker_tests:
+    name: "Modular Docker integration tests (x86_64)"
+    needs: docker_build
+    runs-on: ubuntu-24.04
+    timeout-minutes: 180
+    steps:
+      - id: checkout
+        uses: actions/checkout@v4
+      - id: run_cmake
+        name: Run CMake
+        run: |
+          mkdir build
+          cd build
+          cmake ${DOCKER_CMAKE_FLAGS} ..
+      - name: Download artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: minifi_docker
+          path: build
+      - name: Load Docker image
+        run: |
+          docker load --input ./build/minifi_docker.tar
+      - id: install_deps
+        name: Install dependencies for Docker Verify
+        run: |
+          sudo apt update
+          sudo apt install -y python3-virtualenv
+      - id: free_disk_space
+        run: |
+          # We can gain additional disk space on the Ubuntu runners thanks to these suggestions:
+          # https://github.com/actions/runner-images/issues/2840#issuecomment-790492173
+          # https://github.com/actions/runner-images/issues/2606#issuecomment-772683150
+          sudo rm -rf /usr/share/dotnet
+          sudo rm -rf /usr/local/lib/android
+          sudo rm -rf /opt/ghc
+          sudo rm -rf "/usr/local/share/boost"
+          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+      - id: test
+        name: Docker Verify
+        working-directory: ./build
+        run: make docker-verify-modular
```

Comment:
I think we should document how the modular docker tests should be run using the make target, and also manually a specific feature set. We should also document the required python environment and the required python version. Judging by the syntax used at python 3.10 should be required, for example for using the `|` pipe character when defining type hints. But if we require python 3.10 then maybe we may not need the `typing` package either.